pkg/driver/krun/krun_driver_darwin.go

```diff
@@ -0,0 +1,203 @@
+// SPDX-FileCopyrightText: Copyright The Lima Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package krun
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/lima-vm/lima/v2/pkg/driver"
+	"github.com/lima-vm/lima/v2/pkg/executil"
+	"github.com/lima-vm/lima/v2/pkg/limatype"
+	"github.com/lima-vm/lima/v2/pkg/limatype/filenames"
+	"github.com/lima-vm/lima/v2/pkg/ptr"
+)
+
+type LimaKrunDriver struct {
+	Instance     *limatype.Instance
+	SSHLocalPort int
+
+	krunCmd    *exec.Cmd
```

Comment:
krunCmd is not a good name since the command is krunkit. This is internal detail not visible to users, but it makes the code more confusing.